### ISSUE TITLE
How to grant ADB Wifi for Tasker?

### ISSUE THREAD

**Issue by Rushmed** [joaomgcd/Tasker-Permissions#5]
Hi.

Tried granting ADB Wifi permission for Tasker. But there no ADB Wifi button on the UI.
![image](https://user-images.githubusercontent.com/59259796/143297715-e018b83f-d1fe-4d7e-a4a1-aa70df4e309c.png)
Also tried start as an administrator and of course searche the other app tabs on the top.

**Comment by joaomgcd** [comment]
It's at the top there. :)

**Comment by johnbd2711** [comment]
I want to avoid the Android system prompt when taking a screenshot. I downloaded the tasker_permissions_portable program and ran that on my Windows 7 laptop.  I did the instruction 1 to 4.

When i did Number 5, I just got a request to allow the  laptop access to the phone data, not asking me to allow debugging.

Also, I did not see any Reload button....see attached pic.

Am I doing something wrong?

![Capture](https://user-images.githubusercontent.com/95015561/143395594-8f7ad24d-729f-4766-8762-14c9bcc4fddd.JPG)

**Comment by joaomgcd** [comment]
Did you try clicking the "Reload Devices/Permissions" button at the top there after connecting the phone?

**Comment by johnbd2711** [comment]
Hi Joao

Yes, I tried clicking the "Reload Devices/Permissions" but there was no "Button" under the text. I could highlight the text as shown in this screenshot, but there was no button there. I am on Win 7 if that makes a difference.

Thank you for your help!
John

![20211126_155857](https://user-images.githubusercontent.com/95015561/143535169-f1123d7d-0943-48f9-a517-e8b698431133.jpg)

**Comment by joaomgcd** [comment]
Yeah, sorry, it doesn't look like a button very much, but it really is 😅 What happens when you click it? If nothing, that means that your PC didn't recognize your phone or your phone didn't recognize your PC. Maybe you need some drivers for your particular phone? You should get the debugging prompt on your phone to connect successfully.

**Comment by johnbd2711** [comment]
I have now tried a few times and everything works upto when i plug the phone into the laptop. I get a message asking to allow the laptop to give access to the data on the phone. There is no mention of debugging. My Samsung Galaxy 10 is upto date with software and the previous steps were correctly carried out.

Could it be because the laptop is windows 7?

**Comment by wuschew** [comment]
Hello, I have the same issue like john on a MacOS Monterey 12.1 and Xiaomi Mi10 MIUI Global 12.5.7. All settings are done according your guide, but the app shows still "no devices detected". Reload was tried multiple times without any difference. Any hint? Thanks BR Ives

**Comment by goombah88** [comment]
When I try to run the AppImage file in Fedora linux I get the blank app window for about 5 seconds then it disappears and this error is left in the terminal:

[52575:0225/161252.230944:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.
Trace/breakpoint trap (core dumped)

Tried using the source code (tar.gz) but when I launch index.html it just sits there saying "Loading ADB Permissions..."

In short, no combination of USB debugging and "Use USB for..." seems to make a difference.

**Comment by joaomgcd** [comment]
Are you able to test if you can use ADB manually with that phone?

**Comment by goombah88** [comment]
> Are you able to test if you can use ADB manually with that phone?

Wasn't sure if this was directed to me. Yes, I can see the phone attached:

./adb devices
List of devices attached
<serial #>	unauthorized

**Comment by TungstenE2** [comment]
> It's at the top there. :)

@joaomgcd I have similar issue and your answer does not really help.

None of the boxes is stating 'ADB Wifi permission' although it is mentioned that for easy setting of permission the app should be used.

I granted all permission, but still get the error message that ADB Wifi permission is missing.

So please advice with button exactly is supposed to set permission on Samsung S21.

thx a lot

![grafik](https://user-images.githubusercontent.com/39955685/176995144-66061585-a5ea-44e5-a91b-d58af1438b6f.png)

**Comment by joaomgcd** [comment]
Hi. 

On the app, the option is there in your screenshot below the reload button :) "Enable ADB Wifi in Tasker with port".

In that page the command is also there: "adb tcpip 5555"

Hope this helps!

**Comment by ghbaud** [comment]
Just spent the better part of a day trying to figure this out. Maybe the button should say Enable or Grant or something more explanatory than Confirm.

**Comment by BikeBravak** [comment]
Has Anyone read these?  I'm so trilled to be interested in this.  I haven"t had much luck except for Charging sounds.. I'll keep looking here for some knolage, It will happen. LOL. Good luck folks  And has anyone found the reload button?
